website/docs/docs/quickstarts/dbt-cloud/redshift-qs.md

```diff
@@ -0,0 +1,285 @@
+sidebar_label: "Redshift quickstart"
+---
+For the Redshift project in the quickstart guide, you'll learn how to:
+
+- Set up a Redshift cluster.
+- Load sample data into your Redshift account.
+- Connect dbt Cloud to Redshift.
+- Take a sample query and turn it into a model in your dbt project. A model in dbt is a select statement.
+- Add tests to your models
+- Document your models
+- Schedule a job to run
+
+
+:::tip Videos for you
+You can check out [dbt Fundamentals](https://courses.getdbt.com/courses/fundamentals) for free if you're interested in course learning with videos.
+
+:::
+
+## Prerequisites 
+- You have a  [dbt Cloud account](https://cloud.getdbt.com/). 
+- You have an AWS account with permissions to execute a CloudFormation template to create appropriate roles and a Redshift cluster.
+
+## Create a Redshift cluster
+1. Sign in to your [AWS account](https://signin.aws.amazon.com/console) as a root user or an IAM user depending on your level of access.
+2. Use a CloudFormation template to quickly set up a Redshift cluster. A CloudFormation template is a configuration file that automatically spins up the necessary resources in AWS. [Start a CloudFormation stack](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=dbt-workshop&templateURL=https://tpch-sample-data.s3.amazonaws.com/create-dbtworkshop-infr) and you can refer to the [create-dbtworkshop-infr JSON file](https://github.com/aws-samples/aws-modernization-with-dbtlabs/blob/main/resources/cloudformation/create-dbtworkshop-infr) for more template details.
+
+
+3. Click **Next** for each page until you reach the **Select acknowledgement** checkbox. Select **I acknowledge that AWS CloudFormation might create IAM resources with custom names** and click **Create Stack**.  You should land on the stack page with a CREATE_IN_PROGRESS status.
+
+    <Lightbox src="/img/redshift_tutorial/images/cloud_formation_in_progress.png" title="Cloud Formation in Progress" />
+
+4. When the stack status changes to CREATE_COMPLETE, click the **Outputs** tab on the top to view information that you will use throughout the rest of this guide. Save those credentials for later by keeping this open in a tab.
+
+5. Type `Redshift` in the search bar at the top and click **Amazon Redshift**.
+
+    <Lightbox src="/img/redshift_tutorial/images/go_to_redshift.png" title="Click on Redshift" />
+
+6. Confirm that your new Redshift cluster is listed in **Cluster overview**. Select your new cluster.
+
+<Lightbox src="/img/redshift_tutorial/images/cluster_overview.png" title="Available Redshift Cluster" />
+
+7. Click **Query Data**. You can choose the classic query editor or v2. We will be using the v2 version for the purpose of this guide.
+
+8. You might be asked to Configure account. For the purpose of this sandbox environment, we recommend selecting “Configure account”.
+
+9. Select your cluster from the list and fill out the credentials from the output of the stack:
```

Comment:
When the cluster is selected there is a pop up window presented with multiple options. We should explicitly state in the text to choose "Database user name and password." The screenshot is outdated 
<img width="609" alt="Screenshot 2023-03-06 at 9 30 30 AM" src="https://user-images.githubusercontent.com/60105315/223140183-c98da0a9-dbca-4368-a685-3c87abef28d8.png">